api-reference/10 UI Components/dxStepper/1 Configuration/onOptionChanged.md

```diff
@@ -4,28 +4,143 @@ type: function(e)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed after a UI component property is changed.
 
 ##### param(e): ui/stepper:OptionChangedEvent
-<!-- Description goes here -->
+Information about the event.
 
 ##### field(e.component): {WidgetName}
-<!-- Description goes here -->
+The UI component's instance.
 
 ##### field(e.element): DxElement
-<!-- Description goes here -->
+#include common-ref-elementparam with { element: "UI component" }
 
 ##### field(e.fullName): String
-<!-- Description goes here -->
+The path to the modified property that includes all parent properties.
 
 ##### field(e.name): String
-<!-- Description goes here -->
+The modified property if it belongs to the first level. Otherwise, the first-level property it is nested into.
 
 ##### field(e.previousValue): any
-<!-- Description goes here -->
+The UI component's previous value.
```

Comment:
I want to change this to match the parameter description that follows (`e.value`):

```suggestion
The modified property's previous value.
```